migrations/6_ribbon_eth_covered_call.js

```diff
@@ -16,48 +16,46 @@ module.exports = async function (deployer, network) {
   const networkLookup = network.replace("-fork", "");
   const { admin, owner } = ACCOUNTS[networkLookup];
 
-  // Deploying the ProtocolAdapter
-  // await deployer.deploy(ProtocolAdapterLib);
+  await ProtocolAdapterLib.deployed();
 
-  // await updateDeployedAddresses(
-  //   network,
-  //   "ProtocolAdapterLib",
-  //   ProtocolAdapterLib.address
-  // );
+  await deployer.link(ProtocolAdapterLib, RibbonCoveredCall);
 
-  // await deployer.link(ProtocolAdapterLib, RibbonCoveredCall);
-
-  // // Deploying the logic contract
-  // await deployer.deploy(
-  //   RibbonCoveredCall,
-  //   DEPLOYMENTS[networkLookup].RibbonFactory,
-  //   EXTERNAL_ADDRESSES[networkLookup].assets.weth,
-  //   EXTERNAL_ADDRESSES[networkLookup].assets.usdc,
-  //   EXTERNAL_ADDRESSES[networkLookup].airswapSwap,
-  //   { from: admin }
-  // );
-  // await updateDeployedAddresses(
-  //   network,
-  //   "RibbonETHCoveredCallLogic",
-  //   RibbonCoveredCall.address
-  // );
+  // Deploying the logic contract
+  await deployer.deploy(
+    RibbonCoveredCall,
+    EXTERNAL_ADDRESSES[networkLookup].assets.weth,
+    DEPLOYMENTS[networkLookup].RibbonFactory,
+    EXTERNAL_ADDRESSES[networkLookup].assets.weth,
+    EXTERNAL_ADDRESSES[networkLookup].assets.usdc,
+    EXTERNAL_ADDRESSES[networkLookup].airswapSwap,
+    18,
+    // WETH: 10**18, 10**10 0.0000001
+    // WBTC: 0.000001
+    BigNumber.from("10").pow(BigNumber.from("10")).toString(), // WBTC 10**3
+    { from: admin }
+  );
+  await updateDeployedAddresses(
+    network,
+    "RibbonETHCoveredCallLogic",
+    RibbonCoveredCall.address
+  );
 
   // Deploying the proxy contract
   const initBytes = encodeCall(
     "initialize",
-    ["address", "address", "address", "uint256"],
+    ["address", "address", "uint256", "string", "string"],
     [
-      EXTERNAL_ADDRESSES[networkLookup].assets.weth,
       owner,
       owner,
       parseEther("1000").toString(),
+      "Ribbon ETH-THETA",
+      "rETH-THETA",
```

Comment:
I just checked these [values](https://etherscan.io/token/0x0fabaf48bbf864a3947bdd0ba9d764791a60467a), to be consistent, it's

```
"Ribbon ETH Theta Vault"
"rETH-THETA"
```